tests/virt/cluster/common_templates/centos/test_centos_os_support.py

```diff
@@ -146,16 +146,13 @@ def test_pause_unpause_vm(self, golden_image_vm_object_from_template_multi_cento
             vm=golden_image_vm_object_from_template_multi_centos_multi_storage_scope_class,
         )
 
+    @pytest.mark.migration
     @pytest.mark.polarion("CNV-5841")
     @pytest.mark.dependency(
         name=f"{TESTS_CLASS_NAME}::migrate_vm_and_verify",
         depends=[f"{TESTS_CLASS_NAME}::vm_expose_ssh"],
     )
-    def test_migrate_vm(
-        self,
-        skip_access_mode_rwo_scope_class,
-        golden_image_vm_object_from_template_multi_centos_multi_storage_scope_class,
-    ):
+    def test_migrate_vm(self, golden_image_vm_object_from_template_multi_centos_multi_storage_scope_class):
```

Comment:
So if we pass the `-m migration`, but this test depends on `vm_expose_ssh` which does not have the migration marker, the `test_migrate_vm` will be skipped because `vm_expose_ssh` didn't run?

Reply:
yes. if running with only `migration` marker, all common template tests will be skipped.

Reply:
also when running with `-m 'migration and virt'` - we'll hit the same issue, no? 
then how the `migration` marked can be used? (asking for a friend, because I'm facing the same issue :) )

Reply:
the main usage is to trigger virt-sanity validation when any migration-marked test collected for the run

Reply:
virt-sanity validation will exit the pytest run, so what if you have an RWO storage and you still want to run the other tests on it?

Reply:
if we're talking about t2 lanes, we should always have clusters with rwx storage. that's a req.
for local runs, pass `not migration` (for e.g. `-m virt and not migration`) and sanity check will skip migration validation. or pass `--skip-virt-sanity-check` since you know what you're doing with your local runs :)

Reply:
Ok.. so for storage, this marked will not be usable, as we may run on a non-rwx storages...

Reply:
you can still use the marker (if you want). virt sanity validations are not triggered on storage tests

Reply:
mm not sure how can we make it work when migration test depends on another test

Reply:
renamed marker. maybe now can be of use for storage tests as well

Reply:
It is tricky, but without changes we can't get rid of skips..

you can add `rwx_default_storage` marker only for migration test. And when you run tests, if you know that the default storage class is RWO - you can pass  `-m "..any_marker..  and not rwx_default_storage"` -- in this case migration test will be skipped.

Reply:
after the initial commit (and marker renaming) the current purpose of this new `rwx_default_storage` marker is to mark test with cluster config requirement and trigger appropriate sanity validation.
it's not intended to be used to run only migration tests (can be used to skip those like @dshchedr shown above)